src/formio/components/AddressNL.stories.js

```diff
@@ -0,0 +1,92 @@
+import {expect} from '@storybook/jest';
+import {userEvent, waitFor, within} from '@storybook/testing-library';
+
+import {withUtrechtDocument} from 'story-utils/decorators';
+import {ConfigDecorator} from 'story-utils/decorators';
+import {sleep} from 'utils';
+
+import {mockBRKZaakgerechtigdeInvalidPost} from './AddressNL.mocks';
+import {SingleFormioComponent} from './story-util';
+
+export default {
+  title: 'Form.io components / Custom / Address NL',
+  decorators: [withUtrechtDocument, ConfigDecorator],
+  parameters: {
+    msw: {
+      handlers: [mockBRKZaakgerechtigdeInvalidPost],
+    },
+  },
+  args: {
+    type: 'addressNL',
+    key: 'addressNL',
+    label: 'Address NL',
+    extraComponentProperties: {
+      validate: {
+        required: true,
+      },
+    },
+    evalContext: {},
+  },
+  argTypes: {
+    key: {type: {required: true}},
+    label: {type: {required: true}},
+    type: {table: {disable: true}},
+  },
+};
+
+export const Default = {
+  render: SingleFormioComponent,
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    const postcodeInput = await canvas.findByRole('textbox', {name: 'Postcode'});
+    userEvent.type(postcodeInput, '1234AB');
```

Comment:
```suggestion
    await userEvent.type(postcodeInput, '1234AB');
```
`userEvent` is async, that's probably why you need the sleep calls here.

Reply:
Interaction still fails with `await` :/

Reply:
I'll see if I can figure out things tomorrow! Formio's debouncing does cause some issues here typically :/

Reply:
Yes this is what I was thinking as well